rpcs3/rpcs3qt/instruction_editor_dialog.cpp

```diff
@@ -84,7 +84,8 @@ instruction_editor_dialog::instruction_editor_dialog(QWidget *parent, u32 _pc, c
 void instruction_editor_dialog::updatePreview()
 {
 	bool ok;
-	ulong opcode = t2_instr->text().toULong(&ok, 16);
```

Comment:
but ok is being used
which has now random behaviour if not being used to check for valid number